Keyboard.js

```diff
@@ -8,6 +8,10 @@ define([
 	'./util/misc',
 	'dojo/_base/sniff'
 ], function (declare, aspect, domClass, on, lang, has, miscUtil) {
+	
+	var hasPointer = has("pointer"),
+            hasMSPointer = hasPointer && hasPointer.slice(0, 2) === "MS",
```

Comment:
I think it would be more robust to add something like has-pointer.js to https://github.com/SitePen/dgrid/tree/master/util and make these options all be feature tests so they may be excluded from Dojo builds (e.g. if you don't need IE10 pointer event support). Also would probably suggest an upstream fix to dojo/has...